myapp/forms.py

```diff
@@ -0,0 +1,24 @@
+from django import forms
+from django.contrib.auth.models import User
+from django.contrib.auth.forms import UserCreationForm, PasswordChangeForm, AuthenticationForm
+
+
+class CustomRegistrationFrom(UserCreationForm):
+    username = forms.CharField(label='username', required=False, widget=forms.TextInput(attrs={'class': 'form-control', "placeholder": "Username", "rows": 1, "cols": 22}))
```

Comment:
it's more comfortable to the eye to read short lines, line by line, the benefits:
1. convenience
2. readability
3. speed
4. consistentency with the rest of the code in the project 

try it yourself, and see what's more comfortable :)

Reply:
Okay, I will fix! 👍